demo/server.js

```diff
@@ -43,62 +50,70 @@ initSqlJs().then(SQL => {
   }
 
   // Health
-  app.get('/api/health', (req, res) => {
+  app.get("/api/health", (req, res) => {
     res.send({ healthy: true });
-  })
+  });
```

Comment:
Please don't make changes that aren't relevant to this pull request.

Reply:
Sorry , actually the text got formatted automatically by the Visual code extension, had not made any chages. Let me revert changes .